### ISSUE TITLE
Feature Request: Multiple Windows

### ISSUE THREAD

**Issue by jesse-bentz** [microsoft/terminal#2851]
Multiple tabs are great, but multiple windows with multiple tabs are better. I typically have 4 terminal windows open to deal with daily tasks. I image this has already been requested, but I could not find it. Thanks.

# Proposed technical implementation details (optional)

**Comment by zadjii-msft** [comment]
Do you mean like panes #1000?

**Comment by mdtauk** [comment]
Can't you already open multiple instances?  I'm not sure if tearing out tabs will be supported.

**Comment by joemaller** [comment]
@jesse-bentz please share how to open multiple terminal windows/instances, I haven't been able to figure it out

**Comment by joemaller** [comment]
Oh. Shift click the icon in taskbar. Basic windows function. Sorry I've mostly used Macs for the past few decades.

**Comment by jesse-bentz** [comment]
Yes, exactly. Right click on the icon. Funny, this is also my first windows
machine since like 2001. I didn't intend to do much development

**Comment by jesse-bentz** [comment]
Sorry... continued from last message: I didn't intend to do much
development on it, but with the WSL it actually works well enough.
